android/app/src/main/java/org/mushroomobserver/MainApplication.kt

```diff
@@ -9,7 +9,7 @@ import com.facebook.react.ReactPackage
 import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint.load
 import com.facebook.react.defaults.DefaultReactHost.getDefaultReactHost
 import com.facebook.react.defaults.DefaultReactNativeHost
-import com.facebook.react.flipper.ReactNativeFlipper
+import com.facebook.react.adflipper.ReactNativeFlipper
```

Comment:
95% sure I didn't mean to do this and it was typo; undo and see if anything breaks